packages/app/src/hooks/useBuiltInPlugins.ts

```diff
@@ -1,23 +1,14 @@
+import { plugins } from '@ironclad/rivet-core';
```

Comment:
The code snippet provided is clean, concise, and follows good practices. However, there are a couple of minor improvements that could be made. Here are the recommended changes:

1. **Add comments**: Comments are not present in the code. Adding comments helps in understanding the purpose of the code, making it more maintainable.

```javascript
// Order plugins by label and map to an array of options
const pluginOptions = orderBy(
  Object.entries(plugins).map(([id, plugin]) => ({
    value: id,
    label: plugin.name ?? plugin.id,
  })),
  'label',
);

// Hook to get built-in plugins
export function useBuiltInPlugins() {
  return pluginOptions;
}
```

2. **Error handling**: There is no error handling in place. It would be a good idea to add some error handling mechanism to handle any potential issues.

```javascript
export function useBuiltInPlugins() {
  try {
    return pluginOptions;
  } catch (error) {
    console.error('Failed to get built-in plugins: ', error);
    return [];
  }
}
```

3. **Test Coverage**: Ensure that there are tests covering this function. If not, add them. 

Overall, the code is well written with a clear structure. Great job!